### DISCUSSION TITLE
How to import APL from SIMC

### DISCUSSION THREAD

**Discussion by Sifuedition** [Hekili/hekili#808] [Q&A] [answered]
I've been looking and I found this nice guide from 2015: https://medium.com/@hekili/starting-from-simulationcraft-9d94f6bb512b
Obviously, Hekili has made many changes and improvements since then. There is no longer a simc import tab. I've tried going to Share Priorities and using the import tab there, but, I don't get an editable field where I can paste the simc APL. I am also wanting to make one for Disc Priest, which I assume should be added as a whole new action pack linked to the spec, but, the Create New Pack button just takes me back to the general tab, if it does anything at all.

**Comment by Hekili** [comment, marked as answer]
You can't create a pack for Discipline as I don't have a spec module to support it.

Importing from SimulationCraft isn't recommended; it's the sort of thing that you'd have to know how to troubleshoot before attempting it at all.  I don't really offer support in that regard.

**Reply by Sifuedition** [reply]
Thank you for the quick reply, even if I don't like the answer =)
